index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Paris</title>
+
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles/reset.css">
+  <link rel="stylesheet" href="./styles/weather_widget.css">
+</head>
+<body>
+<aside id="weather-widget">
```

Comment:
I don't think this tag is sutable for widget content. `aside` defines a block on the side of the content for placing categories, links to the archive, tags, ect. If you want to use semantic, apply here `main`, for example.

Reply:
I don't think the` main` tag is appropriate too: theoretically, there may (must) be another content on the page, the main content of the site. I think the widget -- it's kinda like ads or so. Do you think, maybe the `section` tag is suitable here? :thinking: 

In any case, this time I've used `div` (not semantic, but 100% suitable :smile_cat: )
[Fixed](aa28f2d3116e815ad185ea61acdd35a542939e3d)

Reply:
Yeah, you're right, if we fill the widget with smth else: navigation, articles ect, in this case we should reconsider structure and `main` could be less sutable than `aside` in some cases, but in our sitation the widget is the main part of page :) Yeah, `section` would be more sutable here, but `div` is universal, you're right 👌